### PR TITLE
Change to `needs repro` label to update rotation list to include everone.

### PR DESCRIPTION
**Pull request by StellaHuang95**
Based on discussion in retrospective, changing the label to be `needs repro` and adding everyone to the rotation list.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: Bug Report
 about: Create a bug report to help us improve your Python experience in Visual Studio
 title: ''
-labels: needs triage
+labels: needs repro
```
Comment (debonte) [inline .github/ISSUE_TEMPLATE/bug-report.md:5]: Does it need to be in the templates if the action also set the label?

**Review by rchiodo — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/questions-and-help.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name: Questions and Help
 about: Ask questions or request help
 title: ''
-labels: needs triage, question
+labels: ''
```
Comment (debonte) [inline .github/ISSUE_TEMPLATE/questions-and-help.md:5]: Was removing `question` here intentional?

**Review by debonte — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/questions-and-help.md (hunk 2 above)
Comment (StellaHuang95) [reply .github/ISSUE_TEMPLATE/questions-and-help.md:5]: yes, `Needs repro` tells people this issue needs attention and I think that's good enough.

Comment (debonte) [reply .github/ISSUE_TEMPLATE/questions-and-help.md:5]: Ok. I thought maybe `question` was useful for clarifying the user's expectations. It makes me wonder if the question template is needed. Your call. I don't know anything about PTVS issues...

**Review by AdamYoblick — APPROVED** [review]

**Pull request merged**